gneiss/tests/test_model.py

```diff
@@ -5,14 +5,13 @@
 #
 # The full license is in the file COPYING.txt, distributed with this software.
 # ----------------------------------------------------------------------------
-
-
 import pandas as pd
 import statsmodels.formula.api as smf
 from skbio import TreeNode
 from gneiss._model import Model
 import unittest
 import pandas.util.testing as pdt
+import os
```

Comment:
According to [flake8](https://pypi.python.org/pypi/flake8-import-order), I think the order of import should be edited so that standard libraries come first, followed by third-party libraries.